test/langtools/tools/javac/annotations/typeAnnotations/classfile/NestedLambdasCastedTest.java

```diff
@@ -30,7 +30,7 @@
  *          jdk.compiler/com.sun.tools.javac.main
  *          jdk.jdeps/com.sun.tools.javap
  * @build toolbox.ToolBox toolbox.JavapTask
- * @run compile -g NestedLambdasCastedTest.java
+ * @run compile -source 16 -g NestedLambdasCastedTest.java
  * @run main NestedLambdasCastedTest
```

Comment:
To massage the existing tests at some places you are passing -source 16 and at others -release 16. Is there some nuance behind it ?

Reply:
Yes; --release 16 is generally preferred as a cleaner, more complete way of requesting 16-ness from javac. However, use of features like the @modules tag in jtreg can preclude use from --release because of interactions like:

    exporting a package from system module jdk.jdeps is not allowed with --release